main.py

```diff
@@ -40,311 +40,135 @@
 from inferno import *
 from scorescreen import *
 from howtoplay import *
+from button import Button
 
+disp_width = 600
+disp_height = 600
+
+score_path = os.path.join(get_activity_root(), 'data', 'score.pkl')
+
+black = (0, 0, 0)
+white = (255, 255, 255)
 
 class game:
 
     sound = True
 
-    def run(self):
-
+    def __init__(self):
         self.crashed = False
         self.running_mode = None
-        black = (0, 0, 0)
-        white = (255, 255, 255)
-        clock = pygame.time.Clock()
+        self.clock = pygame.time.Clock()
         timer = pygame.time.Clock()
 
-        disp_width = 600
-        disp_height = 600
+        self.gameDisplay = None
+        self.info = None
+
+        self.buttons = []
+
+        self.maxscore = [0, 0, 0, 0, 0, 0]
+    
+    def run_game(self, window, gamenumber):
+        self.running_mode = window()
+        self.running_mode.crashed = self.crashed
+        s = self.running_mode.run(self.gameDisplay, self.info)
 
-        press = 0
+        if scorewindow(self.gameDisplay, s, gamenumber).run():
+            self.run_game(window, gamenumber)
+        
+        self.start()
 
-        info = pygame.display.Info()
-        gameDisplay = pygame.display.get_surface()
+    def show_help(self):
+        a = rules()
+        self.running_mode = a
```

Comment:
Why create `a` and then pass it to `self.running_mode` and then reassign a later to the value of `a.run()`?
Using variable names in this context would be great, `_rules` instead of `a` as it's used only within the function.